middleware/limiter/README.md

```diff
@@ -30,12 +30,13 @@ After you initiate your Fiber app, you can use the following possibilities:
 app.Use(limiter.New())
 
 // Or extend your config for customization
+//Duration of limiter.Config is deprecated, please use Expiration instead. 
```

Comment:
I don't think we need this line, since new users of the middleware aren't going to know `Duration` is even an option.

Reply:
I agree, i just let that of the old users can find what Duration is about, since when i clicked on the warning at vscode terminal, it trow me to one of my .js files. Not to the fiber limiter, well, i will remove the line.